packages/table/src/table.tsx

```diff
@@ -615,6 +622,31 @@ export class Table extends AbstractComponent<ITableProps, ITableState> {
         }
     }
 
+    private maybeScrollTableIntoView() {
+        const { viewportRect } = this.state;
+
+        const maxRowIndex = this.grid.numRows - 1;
+        const maxColIndex = this.grid.numCols - 1;
```

Comment:
i'd just inline these in the consts below, where they are each used exactly once.